### DISCUSSION TITLE
请问哪个插件能做到动态按照分组展示的？

### DISCUSSION THREAD

**Discussion by Anivie** [the1812/Bilibili-Evolved#3809] [提问 / Q&A]
在组件仓库里找了好久也没找到类似的功能，动态过滤器唯独缺了按分类过滤。。。
一直很需要这个功能，不知道为什么b站迟迟不做，之前有个插件能做到但是重装的时候忘记备份油猴了。。。。

**Comment by Geeyun-JY3** [comment]
bilibili时间线筛选——分组查看b站动态：https://greasyfork.org/scripts/396032

不过最近好像失效了。

**Reply by Anivie** [reply]
试了一下，雀食无了。。。

**Comment by kardusenor** [comment]
同求

**Reply by Anivie** [reply]
> 同求

我自己[改了一个](https://gist.github.com/Anivie/ea443081bfd458d41e40fd5c3fa3321e)，但是bug很多，不过勉强能用

**Reply by kardusenor** [reply]
> > 同求
> 
> 我自己[改了一个](https://gist.github.com/Anivie/ea443081bfd458d41e40fd5c3fa3321e)，但是bug很多，不过勉强能用

感谢，我按照我自己的使用习惯[改了一下](https://gist.github.com/kardusenor/f10a0a0ef001d2ca4f151b41e39f017d)
